packages/label-sync/package.json

```diff
@@ -19,7 +19,7 @@
     "start": "probot run ./build/src/label-sync.js",
     "prepare": "npm run compile",
     "pretest": "npm run compile",
-    "test": "cross-env LOG_LEVEL=fatal c8 mocha --timeout=15000 --recursive ./build/test",
+    "test": "cross-env LOG_LEVEL=fatal mocha --timeout=15000 --recursive ./build/test",
```

Comment:
Interesting :) Why? I like having the coverage visible to me while working on a bot.

Reply:
you could add a `coverage` script that has `c8`? It's fine to have the `c8` dependency, we just want the bin to not reset the coverage output, which happens when `c8 foo` is called.